lib/browser/init.ts

```diff
@@ -182,11 +182,31 @@ const { setDefaultApplicationMenu } = require('@electron/internal/browser/defaul
 // menu is set before any user window is created.
 app.once('will-finish-launching', setDefaultApplicationMenu);
 
+const { appCodeLoaded } = process;
+delete process.appCodeLoaded;
```

Comment:
Do we actually need to delete `process.appCodeLoaded` if we don't hit the .mjs condition below (on L190)?

Reply:
We delete it so that app code can't somehow call it, it's just protecting our internals